content/en/docs/concepts/services-networking/service-traffic-policy.md

```diff
@@ -14,7 +14,7 @@ description: >-
 
 <!-- overview -->
 
-{{< feature-state for_k8s_version="v1.23" state="beta" >}}
+{{< feature-state for_k8s_version="v1.26" state="GA" >}}
```

Comment:
stable:
`ERROR 2022/11/15 15:45:38 "GA" is not a valid feature-state, use one of "alpha, beta, deprecated, stable"`

Reply:
Done, thanks